src/htm/types/Types.hpp

```diff
@@ -156,6 +157,10 @@ static const htm::Real32 Epsilon = htm::Real(1e-6);
 #endif
 
 
+  static const UInt SizeInt = sizeof(Int);
```

Comment:
just call sizeof(Int) ?

Reply:
> just call sizeof(Int) ?

Could be. I just figured it'd be best to just make one call to sizeof(), but maybe the difference would be negligible.

Reply:
Oh wait, sizeof() is inlined, right? Nevermind then. I am wrong and you are right, just call the function when needed.